### PR TITLE
Add Aastra 68xxi 5.0.0 plugin

### PR DESCRIPTION
**Pull request by bloom1**
Note: not in testing. Use the plugin repo built by Zuul to test it!

### CODE REVIEW

File: plugins/xivo-aastra/5.0.0/pkgs/pkgs.db [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+b-c: include *.st
+
+[install_aastra-lang]
+a-b: unzip $FILE1
+b-c: cp * i18n/
+
+
+[file_6863i-fw]
+filename: 58015370REV035001018SP16863i.zip
+url: https://www.mitel.com/-/media/58015370rev035001018sp16863i.zip
```
Comment (pc-m) [inline plugins/xivo-aastra/5.0.0/pkgs/pkgs.db]: 404 fir the 6863i fw

File: plugins/xivo-aastra/5.0.0/pkgs/pkgs.db [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+[file_6867i-fw]
+filename: 58015372REV035001018SP16867i.zip
+url: https://www.mitel.com/-/media/58015372rev035001018sp16867i.zip
+size: 44267740
+sha1sum: c5050f4a134a734a82d5aa20d3c1be4d6bc5321e
+
+[file_6869i-fw]
+filename: 58015373rev035001018sp16869i.zip
+url: https://www.mitel.com/-/media/58015373rev035001018sp16869i.zip
```
Comment (pc-m) [inline plugins/xivo-aastra/5.0.0/pkgs/pkgs.db]: 404 on the 6869i

File: plugins/xivo-aastra/5.0.0/pkgs/pkgs.db [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+size: 44968084
+sha1sum: 8ad19807445ac1af8f44002388c054a8bc8755b6
+
+[file_6873i-fw]
+filename: 58015374REV035001018SP16873i.zip
+url: https://www.mitel.com/-/media/58015374rev035001018sp16873i.zip
```
Comment (pc-m) [inline plugins/xivo-aastra/5.0.0/pkgs/pkgs.db]: 404 on the 6873

**Comment by pc-m** [comment]
> Note: not in testing. Use the plugin repo built by Zuul to test it!

That's awesome

File: plugins/xivo-aastra/5.0.0/pkgs/pkgs.db (hunk 1 above)
Comment (bloom1) [reply plugins/xivo-aastra/5.0.0/pkgs/pkgs.db]: Mitel moved the files in a subdirectory during the short time it took for the review! It's fixed now.

**Pull request merged**